datafusion/sqllogictest/test_files/aggregates_topk.slt

```diff
@@ -110,11 +110,11 @@ logical_plan
 physical_plan
 01)SortPreservingMergeExec: [max(traces.timestamp)@1 DESC], fetch=4
 02)--SortExec: TopK(fetch=4), expr=[max(traces.timestamp)@1 DESC], preserve_partitioning=[true]
-03)----AggregateExec: mode=FinalPartitioned, gby=[trace_id@0 as trace_id], aggr=[max(traces.timestamp)], lim=[4]
+03)----AggregateExec: mode=FinalPartitioned, gby=[trace_id@0 as trace_id], aggr=[max(traces.timestamp)]
 04)------CoalesceBatchesExec: target_batch_size=8192
 05)--------RepartitionExec: partitioning=Hash([trace_id@0], 4), input_partitions=4
 06)----------RepartitionExec: partitioning=RoundRobinBatch(4), input_partitions=1
-07)------------AggregateExec: mode=Partial, gby=[trace_id@0 as trace_id], aggr=[max(traces.timestamp)], lim=[4]
+07)------------AggregateExec: mode=Partial, gby=[trace_id@0 as trace_id], aggr=[max(traces.timestamp)]
```

Comment:
I am confused why lim=[4] is deleted when we change VARCHAR to utf8view.

Reply:
I think it is because there is special topk functionality for aggregates: https://github.com/apache/datafusion/blob/main/datafusion/physical-plan/src/aggregates/topk/mod.rs

This special implementation likely doesn't have support for Utf8View

Reply:
Got it, thank you @alamb !